tools/AutomationTestFramework/ManageScripts/Main.ps1

```diff
@@ -0,0 +1,83 @@
+
+. "$PSScriptRoot\GenerateAndUploadTestHelperModule.ps1"
+. "$PSScriptRoot\GenerateAndUploadTestModule.ps1"
+. "$PSScriptRoot\RemoveModules.ps1"
+. "$PSScriptRoot\UploadSignedModules.ps1"
+. "$PSScriptRoot\GenerateRunbook.ps1"
+. "$PSScriptRoot\AccountRunbook.ps1"
+
+<#
+.SYNOPSIS
+Complete Automation Account TEST Framework scenario  
+.PARAMETER i_want_to
+Set of switches to enable or disable certain functionality
+.NOTES
+Prerequisite: Ready to use Azure Automation Account and Azure Storage Account container
+with settings provided in the AzureConfig.ps1 file
+#>
+function Main (
+     [bool]$generateAndUploadTestHelperModule = $true
+    ,[bool]$generateAndUploadTestModule = $true
+    ,[bool]$uploadSignedModules = $true
+    ,[bool]$generateRunbooks = $true
+    ,[bool]$uploadPublishAndStartRunbooks = $true) {
+
+    try {
+
+        $foregroundColor = "DarkMagenta"
+
+        if ($generateAndUploadTestHelperModule) {
+            Write-Host "=== GenerateAndUploadTestHelperModule ========================" -ForegroundColor  $foregroundColor
+            GenerateAndUploadTestHelperModule
+    
+        }
+
+        $srcPath = "$PSScriptRoot\..\..\..\src\"
+        
+        # Project is one of the azure-powershell\src\ResourceManager subfolders
+        $projectList = @('Compute', 'Storage')
+        
+        if ($generateAndUploadTestModule) {
+            Write-Host "=== GenerateAndUploadTestModule ========================" -ForegroundColor $foregroundColor
+            
+            GenerateAndUploadTestsModule `
+                -srcPath $srcPath `
+                -projectList $projectList
+        }
+
+        if ($uploadSignedModules) {
+            Write-Host "=== UploadSignedModuless ========================" -ForegroundColor $foregroundColor
+            
+            RemoveAutomationAccountModules -like "*AzureRm.*"
+            
+            $signedModulesPath = "\\aaptfile01\ADXSDK\PowerShell\2017_10_12_PowerShell\pkgs"
```

Comment:
I think this value should be a passed-in parameter and, if not provided, we shoudl default to the latest drop.  We should be able to get the latest drop by searching on ```20\d\d_\d\d_PowerShell``` and getting the pkgs child item from that directory.

Reply:
done.